core/tests/Test.Microsoft.Identity.Core.UIAutomation/XamarinUITestController.cs

```diff
@@ -43,6 +49,7 @@ public class XamarinUITestController : ITestController
         const int defaultRetryFrequencySec = 1;
         const int defaultPostTimeoutSec = 1;
         const string CSSIDSelector = "[id|={0}]";
+        const string XpathSelector = "//*[text()=\"{0}\"]";
```

Comment:
This is good. One of the recommended ways to search for elements is with the XPath. Im glad you have added this functionality.